packages/blog/src/blueprint/source.js

```diff
@@ -45,7 +45,11 @@ const source = {
 
   // path() determines the final URL path of a Markdown source
   // In `blog` mode, the default format is /YYYY/MM/DD/<slug>
-  path (fileName, { title, published }) {
+  path (fileName, { title, published, meta = {} }) {
```

Comment:
You could destructure the slug already in the argument, no need for using the meta arg here

Reply:
I choose this way because another option would conflict with `const slug` below

Reply:
~~const~~ ;)

But yes, I overlooked that indeed! So ok to leave it I guess